libs/platform/src/lib/components/button/button.component.ts

```diff
@@ -7,9 +7,16 @@ import {
     ElementRef
 } from '@angular/core';
 
-export type ButtonType = 'standard' | 'positive' | 'medium' | 'negative' | 'toolbar' | 'main';
-export type ButtonOptions = 'light' | 'emphasized' | '';
-
+export type ButtonType =
+    | ''
+    | 'standard'
+    | 'positive'
+    | 'negative'
+    | 'attention'
+    | 'ghost'
+    | 'transparent'
+    | 'emphasized'
+    | 'menu';
 @Component({
```

Comment:
Also we are creating BaseComponent connect to Loki and Deepak, so you can rename some of your bindings and extend it from base component.

Reply:
Sure will do that, but Base component work is in progress and it is not available in master yet. 

Do you want me to wait until it is merged because PR won't be kept open for a long time it will get closed automatically after 20days. 

Or merge this now and update the code and raise one more PR once the base component is available in master by this I can proceed with app-framework documentation changes.